_documentation/user/flags/seer.md

```diff
@@ -5,4 +5,14 @@ type: Good
 summary: See stealthed, cloaked and masquerading tanks as normal.
 ---
 
-...
+The Seer (SE) flag allows the player to see the tanks holding the [Stealth (ST)](../stealth/), [Cloaking (CL)](../cloaking/), or [Masquerade (MQ)](../masquerade/) flags as normal.
+
+Its power is therefore to nullify the effects of those three flags for the holder, rather than bringing out a unique power-up itself.
```

Comment:
```suggestion
Its power is therefore to nullify the effects of those flags for the holder, rather than bringing out a unique power-up itself.
```

While unlikely this would happen any time soon, I'd prefer to avoid specifying set limits as new flags could be introduced in future versions of the game

Reply:
@allejo yeah not mentioning the count would make it more flexible, agreed.